task/sql-tasks.js

```diff
@@ -199,7 +370,34 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            sum(IF(monthName = 'January', total, 0)) AS January,
+            sum(IF(monthName = 'February', total, 0)) AS February,
+            sum(IF(monthName = 'March', total, 0)) AS March,
+            sum(IF(monthName = 'April', total, 0)) AS April,
+            sum(IF(monthName = 'May', total, 0)) AS May,
+            sum(IF(monthName = 'June', total, 0)) AS June,
+            sum(IF(monthName = 'July', total, 0)) AS July,
+            sum(IF(monthName = 'August', total, 0)) AS August,
+            sum(IF(monthName = 'September', total, 0)) AS September,
+            sum(IF(monthName = 'October', total, 0)) AS October,
+            sum(IF(monthName = 'November', total, 0)) AS November,
+            sum(IF(monthName = 'December', total, 0)) AS December
+        FROM
+            (
+                SELECT
+                    count(DISTINCT OrderID) AS total,
+                    MONTHNAME(OrderDate) AS monthName
+                FROM
+                    Orders
+                WHERE
+                    YEAR(OrderDate) = 1997
+                GROUP BY
+                    2
+            ) query
+    `)

```

Comment:
Please, rework query with `count (if (condition))`
And this task can be solved with just one SELECT